### PR TITLE
feat: update cli

### PR DESCRIPTION
**Pull request by berezovskycom**
Fixes an Error in node 14, regarding fs module.
Tested on node 10, 12, 14 and it works fine.
Issue #276

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
   },
   "dependencies": {
     "@babel/runtime-corejs3": "^7.9.6",
-    "@elasticprojects/abstract-cli": "^4.3.0",
+    "@elasticprojects/abstract-cli": "^11.1.1-dev.20200622100140",
```
Comment (tommoor) [inline package.json:71]: Can we use a non-dev release, `11.1.1` should contain the fix – do you know which PR on the backend addressed the issue?

Comment (berezovskycom) [reply package.json:71]: Oh, yeah, sure, my bad.
[#4854](https://github.com/goabstract/projects/pull/4854)
Does 11.1.1 contain the fix?

Comment (tommoor) [reply package.json:71]: It does

**Comment by berezovskycom** [comment]
@tommoor had to downgrade to `11.1.0`, since `11.1.1 `is unavailable via npm (works anyway).
Screenshot 2020-06-24 at 15 09 07

**Pull request merged**